src/components/sub/menu/menu-bar.jsx

```diff
@@ -1,134 +1,145 @@
     },
     {
-      icon: 'code-view',
-      title: 'Code',
+      icon: RiCodeView,
+      title: "Code",
       action: () => editor.chain().focus().toggleCode().run(),
-      isActive: () => editor.isActive('code'),
-    },
-    {
-      icon: 'mark-pen-line',
-      title: 'Highlight',
-      action: () => editor.chain().focus().toggleHighlight().run(),
-      isActive: () => editor.isActive('highlight'),
-    },
-    {
-      type: 'divider',
-    },
-    {
-      icon: 'h-1',
-      title: 'Heading 1',
-      action: () => editor.chain().focus().toggleHeading({ level: 1 }).run(),
-      isActive: () => editor.isActive('heading', { level: 1 }),
-    },
-    {
-      icon: 'h-2',
-      title: 'Heading 2',
-      action: () => editor.chain().focus().toggleHeading({ level: 2 }).run(),
-      isActive: () => editor.isActive('heading', { level: 2 }),
-    },
-    {
-      icon: 'paragraph',
-      title: 'Paragraph',
-      action: () => editor.chain().focus().setParagraph().run(),
-      isActive: () => editor.isActive('paragraph'),
-    },
-    {
-      icon: 'list-unordered',
-      title: 'Bullet list',
-      action: () => editor.chain().focus().toggleBulletList().run(),
-      isActive: () => editor.isActive('bulletList'),
-    },
-    {
-      icon: 'list-ordered',
-      title: 'Ordered list',
-      action: () => editor.chain().focus().toggleOrderedList().run(),
-      isActive: () => editor.isActive('orderedList'),
-    },
-    {
-      icon: 'list-check-2',
-      title: 'Task list',
-      action: () => editor.chain().focus().toggleTaskList().run(),
-      isActive: () => editor.isActive('taskList'),
-    },
-    {
-      icon: 'code-box-line',
-      title: 'Code block',
-      action: () => editor.chain().focus().toggleCodeBlock().run(),
-      isActive: () => editor.isActive('codeBlock'),
-    },
-    {
-      type: 'divider',
-    },
-    {
-      icon: 'double-quotes-l',
-      title: 'Blockquote',
-      action: () => editor.chain().focus().toggleBlockquote().run(),
-      isActive: () => editor.isActive('blockquote'),
-    },
-    {
-      icon: 'separator',
-      title: 'Horizontal rule',
-      action: () => editor.chain().focus().setHorizontalRule().run(),
-    },
-    {
-      type: 'divider',
-    },
-    {
-      icon: 'text-wrap',
-      title: 'Hard break',
-      action: () => editor.chain().focus().setHardBreak().run(),
-    },
-    {
-      icon: 'format-clear',
-      title: 'Clear format',
-      action: () => editor.chain().focus().clearNodes().unsetAllMarks()
-        .run(),
-    },
-    {
-      type: 'divider',
-    },
-    {
-      icon: 'arrow-go-back-line',
-      title: 'Undo',
-      action: () => editor.chain().focus().undo().run(),
-    },
-    {
-      icon: 'arrow-go-forward-line',
-      title: 'Redo',
-      action: () => editor.chain().focus().redo().run(),
-    },
-  ]
+      isActive: () => editor.isActive("code"),
+    },
+    // {
+    //   icon: "mark-pen-line",
```

Comment:
you can uncomment these and apply the same fix for icons as I did above. I didn't want to do them all so I just commented those I didn't do.

the first 4 are my POC for this fix